modules/admin_manual/pages/release_notes.adoc

```diff
@@ -60,7 +60,7 @@ If you are still running a PHP version < 7.2, you must upgrade PHP before upgrad
 
 Summarizing, ownCloud Server 10.5 supports the PHP versions **7.2, 7.3 and 7.4**.
 
-TIP: See the xref:installation/system_requirements.adoc#officially-recommended-supported-options[system requirements in the ownCloud Documentation] for more information.
+TIP: See the xref:installation/system_requirements.adoc#officially-recommended-supported-options[system requirements in the ownCloud Documentation] for the recommended PHP version and for more information..
```

Comment:
@pmaier1 there is a double dot at the end of the sentence. One dot needs to be removed.